<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWindow</class>
 <widget class="QDialog" name="RegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>328</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>272</width>
    <height>328</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* === Глобальний стиль вікна === */
QDialog, QWidget {
    background-color: #f2f2f2;        /* світлий фон */
    color: #202020;                   /* основний текст */
    font-family: &quot;Stencil&quot;;
    font-size: 15px;
    letter-spacing: 1px;
}

/* === Заголовки === */
QLabel#labelTitle {
    font-size: 40px;
    font-weight: bold;
    color: #222222;
}

/* === Звичайні написи === */
QLabel {
    color: #303030;
    font-weight: 400;
}

/* === Поля вводу === */
QLineEdit {
    background-color: #ffffff;
    border: 2px solid #bbbbbb;
    border-radius: 6px;
    padding: 6px 10px;
    font-family: &quot;Segoue UI&quot;;
    color: #202020;
    selection-background-color: #00cc88;
    selection-color: white;
}

QLineEdit:focus {
    border: 2px solid #00cc88;
    background-color: #fcfcfc;
}

/* === Статус-текст (labelStatus) === */
QLabel#labelStatus {
    font-size: 13px;
    margin-top: 4px;
    font-weight: italic;
}

/* зелений текст — успіх */
QLabel#labelStatus[styleSheet*=&quot;green&quot;] {
    color: #009966;
}

/* червоний текст — помилка */
QLabel#labelStatus[styleSheet*=&quot;red&quot;] {
    color: #cc0033;
}

/* === Кнопки === */
QPushButton {
    background-color: #00cc88;
    color: white;
    border: none;
    border-radius: 8px;
    padding: 8px 18px;
    font-family: &quot;Stencil&quot;;
    font-weight: bold;
    text-transform: uppercase;
    letter-spacing: 1px;
}

QPushButton:hover {
    background-color: #00b377;
    box-shadow: 0 0 8px #00cc88;
}

QPushButton:pressed {
    background-color: #009966;
    transform: scale(0.97);
}

QPushButton:disabled {
    background-color: #cccccc;
    color: #777777;
}

/* === Чекбокс (Показати пароль) === */
QCheckBox {
    color: #303030;
    spacing: 6px;
}

QCheckBox::indicator {
    width: 16px;
    height: 16px;
    border: 2px solid #888888;
    border-radius: 4px;
    background-color: white;
}

QCheckBox::indicator:checked {
    background-color: #00cc88;
    border: 2px solid #00cc88;
}

QCheckBox:hover {
    color: #00aa66;
}

/* === Скролбар (на випадок, якщо є поле з прокруткою) === */
QScrollBar:vertical {
    border: none;
    background: #e0e0e0;
    width: 8px;
    margin: 0;
    border-radius: 4px;
}

QScrollBar::handle:vertical {
    background: #00cc88;
    border-radius: 4px;
    min-height: 20px;
}

QScrollBar::handle:vertical:hover {
    background: #00aa66;
}

QScrollBar::add-line:vertical,
QScrollBar::sub-line:vertical {
    height: 0;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="icon">
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources.qrc">:/todoApp/resources/icons/to-do-list.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="indent">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelTitle">
       <property name="font">
        <font>
         <family>Stencil</family>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Register</string>
       </property>
       <property name="pixmap">
        <pixmap>../resources/icons/to-do-list.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="labelLogin">
       <property name="text">
        <string>Username:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditUsername">
       <property name="maximumSize">
        <size>
         <width>450</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoue UI</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="placeholderText">
        <string>Username</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="labelPassword1">
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPassword">
       <property name="maximumSize">
        <size>
         <width>450</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::EchoMode::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Password</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="labelPassword2">
       <property name="text">
        <string>Confirm password:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditConfirmPassword">
       <property name="maximumSize">
        <size>
         <width>450</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::EchoMode::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Confirm password</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelStatus">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>LABELSTATUT</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnRegister">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
